java/core/BUILD

```diff
@@ -118,6 +118,18 @@ java_library(
     srcs = LITE_SRCS,
 )
 
+java_library(
```

Comment:
This shouldn't be needed, kt rules should be able to rely on `core` and `lite`, I think this is identical, and generally its a bad practice to have srcs in multiple targets as this proposes. If you can't depend on java_export in a kt_ that is likely a bug we need to fix.

Reply:
Internally java_export uses java_library, so it should be OK: https://github.com/bazelbuild/rules_jvm_external/blob/master/private/rules/java_export.bzl#L66

Reply:
Ok changed to now rely on existing rules and removed this new rule